client/src/components/lists/MembershipOption/option.tsx

```diff
@@ -28,14 +32,20 @@ const styles = (theme: Theme) =>
   });
 
 interface Props extends WithStyles<typeof styles> {
+  theme: Theme;
   option: MembershipOption;
   isLast?: boolean;
 }
 
-function Option({ classes, option, isLast }: Props) {
+function Option({ classes, theme, option, isLast }: Props) {
+  const [showOptions, setShowOptions] = useState(false);
   const [showDescription, setShowDescription] = useState(false);
   const { setOption, setIsOpen } = useContext(AuthPanelContext);
 
+  useEffect(() => {
+    setTimeout(() => setShowOptions(true), 1000);
+  });
```

Comment:
I think this boy will run on every render since u didn't pass a dependency array

```suggestion
  useEffect(() => {
    setTimeout(() => setShowOptions(true), 1000);
  }, []);
```

An empty deps array will make it run only on first render 💯